### PR TITLE
Add kubeflow bundle for 1.6 using tracks and edge risk

### PR DESCRIPTION
**Pull request by ca-scribner**
This PR adds the bundle files for the kubeflow 1.6 release that pull from each charm's invidual track in the edge risk.

### CODE REVIEW

File: releases/1.6/kubeflow-bundle.yaml [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+    charm: metacontroller-operator
+    channel: 0.3/edge
+    scale: 1
+    trust: true
+    _github_repo_name: metacontroller-operator
+  mlmd:
+    charm: mlmd
+    channel: 1.0/edge
+    scale: 1
+    _github_repo_name: mlmd-operator
```
Comment (DomFleischmann) [inline releases/1.6/kubeflow-bundle.yaml:147]: I thought we were going to remove mlmd for this release.

File: releases/1.6/kubeflow-bundle.yaml [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+    charm: dex-auth
+    channel: 2.31/edge
+    scale: 1
+    trust: true
+    _github_repo_name: dex-auth-operator
+  envoy:
+    charm: envoy
+    channel: 1.20/edge
+    scale: 1
+    _github_repo_name: envoy-operator
```
Comment (DomFleischmann) [inline releases/1.6/kubeflow-bundle.yaml:29]: Same as mlmd

**Review by DomFleischmann — COMMENTED** [review]
Looks good besides those comments

File: releases/1.6/kubeflow-bundle.yaml (hunk 1 above)
Comment (ca-scribner) [reply releases/1.6/kubeflow-bundle.yaml]: Removed

File: releases/1.6/kubeflow-bundle.yaml (hunk 2 above)
Comment (ca-scribner) [reply releases/1.6/kubeflow-bundle.yaml]: Removed

File: releases/1.6/kubeflow-bundle.yaml [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+    channel: 1.6/edge
+    scale: 1
+    trust: true
+    _github_repo_name: kubeflow-roles-operator
+  kubeflow-volumes:
+    charm: kubeflow-volumes
+    channel: 1.6/edge
+    scale: 1
+    _github_repo_name: kubeflow-volumes-operator
+  metacontroller-operator:
```
Comment (DnPlas) [inline releases/1.6/kubeflow-bundle.yaml:132]: metacontroller is [2.0.4](https://github.com/canonical/metacontroller-operator/commit/584cf1d9d79af1550ef64e9e5744dc2f3faa445d)

**Review by DnPlas — APPROVED** [review]
Thanks @ca-scribner, except for metacontroller-operator, but we can figure it out later.

**Pull request merged**